src/com/android/volley/toolbox/ImageLoader.java

```diff
@@ -196,7 +220,7 @@ public ImageContainer get(String requestUrl, ImageListener imageListener,
      *     the currently available image (default if remote is not loaded).
      */
     public ImageContainer get(String requestUrl, ImageListener imageListener,
```

Comment:
can you add documentation on backgroundFetch (just say to pass false for normal requests, true if it's to pre-fetch something in the background). and on doublerespond if you'd like (but that is more clear)